### PR TITLE
[chore] Update msgpack

### PR DESCRIPTION
**Pull request by patrick-ogrady**
Closes: #243

This PR updates `msgpack` to the latest release and resolves breaking API changes.

### CODE REVIEW

File: storage/encoder.go [hunk 1]
```diff
@@ -84,7 +88,7 @@ func NewEncoder(
 
 func getEncoder(w io.Writer) *msgpack.Encoder {
 	enc := msgpack.NewEncoder(w)
-	enc.UseJSONTag(true)
+	enc.SetCustomStructTag(jsonTag)
```
Comment (patrick-ogrady) [inline storage/encoder.go:91]: the `msgpack` api changed 😢

**Review by septerr — APPROVED** [review]

**Comment by heimdall-asguard** [comment]
Review Error for septerr @ 2020-11-23 18:10:40 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by septerr — APPROVED** [review]

**Comment by heimdall-asguard** [comment]
Review Error for septerr @ 2020-11-23 18:14:31 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by bobg — APPROVED** [review]

**Review by septerr — APPROVED** [review]

**Comment by heimdall-asguard** [comment]
Review Error for septerr @ 2020-11-23 18:23:47 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Pull request merged**